wdio.conf.js

```diff
@@ -1,312 +0,0 @@
-require('dotenv').config();
-require('babel-register');
-
-const os = require('os');
-
-// eslint-disable-next-line prefer-destructuring
-const argv = require('yargs').argv;
-
-const uuid = require('uuid');
-
-const {inject} = require('./scripts/tests/openh264');
-const beforeSuite = require('./scripts/tests/beforeSuite');
-
-
-const browser = process.env.BROWSER || 'chrome';
-const platform = process.env.PLATFORM || 'mac 10.12';
-const tunnelId = uuid.v4();
-const port = process.env.PORT || 4567;
-const {suite} = argv || 'integration';
-const screenResolutionMac = '1920x1440';
-const screenResolutionWin = '1920x1080';
-const screenResolution = platform.toLowerCase().includes('os x') || platform === 'darwin' || platform.includes('mac') ? screenResolutionMac : screenResolutionWin;
```

Comment:
This was not brought over to the new wdio

Reply:
Good catch